custom_components/senec/pysenec_ha/__init__.py

```diff
@@ -1602,6 +1658,11 @@ async def update_get_systems(self, a_plant_number: int):
                 self._isAuthenticated = False
                 await self.authenticate(doUpdate=False, throw401=False)
 
+    @property
+    def spare_capacity(self) -> float:
+        if hasattr(self, '_spare_capacity'):
+            return str(self._spare_capacity)
```

Comment:
here I would use again int(self._spare_capacity) and also use the return type int (and not float)

Reply:
Updated and successfully tested.